Composer/packages/client/src/components/ManageQNA/ManageQNA.tsx

```diff
@@ -518,14 +518,14 @@ export const ManageQNA = (props: ManageQNAProps) => {
       <Dialog
         dialogContentProps={{
           type: DialogType.normal,
-          title: currentPage === 2 ? formatMessage('Create new QnA resources') : formatMessage('Select QnA keys'),
+          title: currentPage === 2 ? formatMessage('Create new QnA resource') : formatMessage('Select QnA keys'),
         }}
         hidden={props.hidden}
         minWidth={480}
         modalProps={{
           isBlocking: true,
         }}
-        onDismiss={props.onDismiss}
+        onDismiss={!loadingQNA ? props.onDismiss : () => {}}
```

Comment:
Same here; if you're writing a `?:` thing with a "not" in the condition, just remove the "not" and swap the two results.